lib/transformations/index.js

```diff
@@ -37,7 +37,7 @@ function transform(source, transforms, options) {
 		quote: 'single'
 	}, options);
 	transforms = transforms || Object.keys(transformations).map(k => transformations[k]);
-	transforms.forEach(f => f(jscodeshift, ast));
+	transforms.forEach(f => f(jscodeshift, ast, source));
```

Comment:
In order to print the code block: https://github.com/webpack/webpack-cli/pull/91/files#diff-33fa12ea113366fa6b41cce3aeb9bf6dR29